server/app/views/admin/TranslationFormView.java

```diff
@@ -96,7 +103,13 @@ protected final FormTag renderTranslationForm(
                             "Save %s updates",
                             locale.getDisplayLanguage(LocalizedStrings.DEFAULT_LOCALE)))
                     .withId("update-localizations-button")
-                    .withClasses(ButtonStyles.SOLID_BLUE));
+                    .withClasses(ButtonStyles.SOLID_BLUE),
+                div(a().withHref(redirectUrl)
+                        .with(
+                            button("Back")
+                                .withId("back-to-edit")
```

Comment:
Since the redirectUrl is going to the list screen the Id name is confusing. It would make more sense to call it `back-to-list-button`.

Reply:
done!